esdoc.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
With Typescript, we can have better code documentation. `esdoc` was breaking anyway.